Frontend/ui-customer/src/Helpers/Utils.js

```diff
@@ -522,7 +522,7 @@ export const contactPermission = function () {
 }
 
 export const performHapticFeedback = function () {
-  if(window.JBridge.performHapticFeedback){
+  if(window.JBridge.performHapticFeedback && (window.JBridge.getAndroidVersion >= 8) ){
```

Comment:
@Rohit4997 getAndroidVersion returns the SDK version of the device. So if you want to refer to android version 8 then here you should mention the SDK version which is 26 in this case

Reply:
@MercyQueen  can you check now?